website/addons/googledrive/tests/test_views.py

```diff
@@ -5,183 +5,83 @@
 
 from nose.tools import *  # noqa
 from framework.auth import Auth
-from website.util import api_url_for
+from website.addons.googledrive.client import GoogleDriveClient
+from website.util import api_url_for, web_url_for
 from tests.base import OsfTestCase, assert_is_redirect
-from tests.factories import AuthUserFactory, ProjectFactory
+from tests.factories import AuthUserFactory, ProjectFactory, ExternalAccountFactory
 
 from website.addons.googledrive.tests.utils import mock_folders
```

Comment:
I agree regarding renaming `utils` that it would make more sense to name it `data` or similar for GD's case, but test data for each addon is contained within a `utils.py`, so consistency might be better than sense.

I will rename `mock_folders`